### PR TITLE
Add known issue for vendor update not working

### PR DESCRIPTION
**Pull request by sciencewhiz**

### CODE REVIEW

File: source/docs/yearly-overview/known-issues.rst [hunk 1]
```diff
@@ -8,6 +8,13 @@ This article details known issues (and workarounds) for FRC\ |reg| Control Syste
 Open Issues
 -----------
 
+VS Code Vendor Check for Updates Broken
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Issue:** Checking for vendor library updates online results in the following error message: `Command 'WPILib: Manage Vendor Libraries' resulted in an error (Only absolute URLs are supported)`. This is caused by a bug in the VS Code extension related to the way the WPILib Command library vendordeps were created. This issue will be fixed in the next WPILib release.
+
+**Workaround:** If you aren't using either the new or old Command vendordep, remove them from your project.
```
[the remark this answers is not in the document]
Comment (sciencewhiz) [reply source/docs/yearly-overview/known-issues.rst:16]: If you don't use them, why would you need to re-add them?

[the remark this answers is not in the document]
Comment (Daltz333) [reply source/docs/yearly-overview/known-issues.rst:16]: Oh sorry, I misread. Either way, I think this should get adjusted for the teams that *are* using commandbased.

[the remark this answers is not in the document]
Comment (sciencewhiz) [reply source/docs/yearly-overview/known-issues.rst:16]: Is there a workaround besides don't use the feature?

[the remark this answers is not in the document]
Comment (Daltz333) [reply source/docs/yearly-overview/known-issues.rst:16]: You can delete, update vendordep, then readd the command json if you are a commandbased team.

[the remark this answers is not in the document]
Comment (NoahAndrews) [reply source/docs/yearly-overview/known-issues.rst:16]: Side note: "re-add" is much clearer than "readd", which I always think is saying "read" at first.

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**